### PR TITLE
Add doc on mining companion

### PR DESCRIPTION
**Pull request by killerwhile**

### CODE REVIEW

File: Mining-companion.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+So if you want to centralize the mining block rewards
+to a normal wallet (with a single address), the mining companion will help you automating this task!
+
+# The mining companion
+
+The mining companion is a lightweight, standalone process which, in a nutshell,
+will connect to a Alephium full node to transfer the block rewards from all the addresses of your mining wallet
+to a normal, single address wallet.
+This transfer operation happens at a regular frequency, for instance every 15 minutes.
+At the triggerr time, if the available (unlocked) amount is bigger than a certain threshold,
```
Comment (polarker) [inline Mining-companion.md]: typo: `triggerr`

**Review by polarker — APPROVED** [review]

File: Mining-companion.md (hunk 1 above)
Comment (killerwhile) [reply Mining-companion.md]: Good catch, thanks!

File: Mining-companion.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+Mining companion for your Alephium full node
+====
+
+Mining is the activity of securing the blockchain providing compute power, while getting
+mining block rewards in counterpart. Every block you mine has a reward attached to it,
+and this reward is sent to the mining wallet that has successfully mined the block.
+
+On the Alephium blockchain, a mining wallet has multiple addresses referring to it,
+precisely one address per chain in the blockchain.
+Alephium testnet has currently 4 chains, so a mining wallet for the testnet has 4 addresses.
```
Comment (tdroxler) [inline Mining-companion.md]: More precisely we have 4 groups, which gives G*G => 16 chains. I don't know how we can explain it in one line @polarker ?

File: Mining-companion.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    labels:
+      - org.label-schema.group=alephium
+```
+
+In the snippet above, replace `YOUR WALLET ADDRESS HERE!!` by the address of you normal wallet, for instance
+`14FGvG61tqzXXYi6UKtzjozMjxCArF1beoU4ogUqM2pAC` and make sure `ALEPHIUM_ENDPOINT` points to
+your Alephium full node endpoint. Using `http://localhost:12973` here is always wrong!
+
+This is it, all the other default configuration options are safe to run. All the configuration options
+are listed in the [README file of the project source code](https://github.com/touilleio/alephium-mining-companion.
```
Comment (tdroxler) [inline Mining-companion.md:60]: it's not a public repo yet right?

File: Mining-companion.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+You can then simply start the process, using `docker-compose up -d mining-companion`, and it will
+start transferring the block rewards to the address you put in the `TRANSFER_ADDRESS` variable.
+
+Disclaimer: If you put a wrong address in `TRANSFER_ADDRESS`, your ALPH rewards can be not usable anymore, or not by you!
```
Comment (tdroxler) [inline Mining-companion.md]: maybe write it has **DISCLAIMER** ^^

**Review by tdroxler — APPROVED** [review]
minor comments, LGTM

File: Mining-companion.md (hunk 2 above)
Comment (killerwhile) [reply Mining-companion.md]: true! thanks for the clarifications!

File: Mining-companion.md (hunk 3 above)
Comment (killerwhile) [reply Mining-companion.md:60]: Good point, I forget to pass it public. It is now.

**Review by tdroxler — APPROVED** [review]

**Pull request merged**